agir/events/components/eventPage/EventDescriptionCard.js

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+
+import { DateTime } from "luxon";
+import Button from "@agir/front/genericComponents/Button";
+import Collapsible from "@agir/front/genericComponents/Collapsible.js";
+import Card from "../../../front/components/genericComponents/Card";
+import Spacer from "../../../front/components/genericComponents/Spacer";
+
+import style from "@agir/front/genericComponents/_variables.scss";
+
+const DescriptionSection = styled.div`
+  margin: 0;
+
+  &:empty {
+    margin: 0;
+  }
+
+  & + & {
+    margin-top: 1rem;
+  }
+
+  & > * {
+    margin-top: 0;
+    margin-bottom: 1rem;
+
+    &:empty,
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  img {
+    margin: 10px 0px;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  height: 48px;
+  font-size: 16px;
+  font-weight: 500;
+  border-radius: 0.5rem;
```

Comment:
On a une variable maintenant pour la nouvelle valeur de border-radius :
```suggestion
  border-radius: ${style.borderRadius};
```